plugin/evm/vm_test.go

```diff
@@ -281,6 +275,11 @@ func testBuildEthTxBlock(t *testing.T, scheme string) {
 		t.Fatalf("Found unexpected blkID for parent of blk2")
 	}
 
+	// Close the vm and all databases
+	if err := vm.Shutdown(context.Background()); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
This close seems to be needed because of racy behavior with the async acceptor and firewood.